### DISCUSSION TITLE
supportable SemanticTokenModifier at Scala 2

### DISCUSSION THREAD

**Discussion by ShintaroSasaki** [ShintaroSasaki/metals#7] [Q&A] [answered]
For current , the following modifiers are server-capable  as default. 
But actually, checked ones will be only provided.  Does it seem appropriate ?
There is  a brief description of each modifier in the middle of [this page](https://code.visualstudio.com/api/language-extensions/semantic-highlight-guide).
- [ ] Declaration
- [ ] Definition
- [x] Readonly
- [ ] Static
- [ ] Deprecated
- [x] Abstract
- [ ] Async
- [ ] Modification
- [ ] Documentation
- [ ] DefaultLibrary

**Comment by tgodzik** [comment, marked as answer]
> Deprecated

We should use:
`sym.isDeprecated`

> Static

Maybe can used via:
`sym.owner.isModule`


We could probably also use Declaration and Definition, but that might be more compelx so we can leave it for later.

There rest don't seem usable for us right now.
